.github/SUPPORT.md

```diff
@@ -0,0 +1,12 @@
+
+# Gutenberg Project Support
+
+Welcome to WordPress' Gutenberg project! We hope you join us in creating the future platform for publishing; all are welcome here.
```

Comment:
The first sentence reads a bit awkwardly right now so I might rephrase particularly for the sake of non English speakers.  Ideas: 

- "Welcome to the Gutenberg Project, a focus of the wider WordPress project." 
- "Welcome to Gutenberg, a WordPress project."

Reply:
I like the second one better, 👍
I grabbed it from the [CONTRIBUTING.md page](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md), so we may want to update there too.